db/sample_data_files/mortality_tracking/example_of_mortality_tracking_data.csv

```diff
@@ -1,2234 +1,1966 @@
-Mortality_date,Cohort,SHL number,Spawning_date,Shell_box,Shell_container,Animal_location,# Morts,Approximation?,Processed by SHL?,Initials,Tag(s),Comments,,,,,,,,,,,,,,,
-9/11/16,2001 Cohort ,SF08-26,4/23/01,SHL,SHL,Broodstock Tank 3,1,,Y,EH,Green_323,"SF08-26-19. Shrunken, Blythe processed",,,,,,,,,,,,,,,

```

Comment:
@edwinthinks just to confirm, all you changed here was the removal of trailing `,`?